templates/email_relation_request.html

```diff
@@ -0,0 +1,9 @@
+<h2>Hello, {{ user_recipient_name }}</h2>
+<p>You have new relation request!</p>
+<br>
+<p>{{ user_sender_name }} has sent you a relation request. He would like to be your {{ sender_role }}. Note from {{ user_sender_name }}: {{ note }}</p>
```

Comment:
```suggestion
<p>{{ user_sender_name }} has sent you a relation request. This member would like to be your {{ sender_role }}. Note from {{ user_sender_name }}: {{ note }}</p>
```

Reply:
Please avoid using pronouns when you don't know the gender of the user that sent the request.
Also can you put the notes in a separate paragraph, because the notes could be a long string.

Reply:
Done! :D